source-oracle-flashback/source_oracle_flashback/resources.py

```diff
@@ -96,11 +96,9 @@ async def validate_flashback(
                 log.warn("We recommend guaranteeing retention of the undo tablespace. See go.estuary.dev/source-oracle for more information.")
 
 
-async def all_resources(
-    log: Logger, http: HTTPMixin, config: EndpointConfig, pool: oracledb.AsyncConnectionPool, is_rds: bool,
-) -> list[common.Resource]:
-    resources_list = []
-
+async def get_tables(
```

Comment:
I can't actually find anywhere that this `get_tables` function is being used, am I missing something?

Reply:
Forgot to remove this after a refactor 🤦🏽